charts/victoria-metrics-anomaly/CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## Next release
 
-- TODO
+- Add ability to render full vmanomaly config via `config` parameter.
```

Comment:
It is better to update this item to clearly state that this is a breaking change and `config` updates will be required for upgrade.
Currently, it seems like this only adds an additional field/option to provide full configuration without breaking compatibility.